demo/malloy-demo-composer/public/index.html

```diff
@@ -28,14 +28,14 @@
       rel="preload"
       href="fonts/GoogleSans[GRAD,opsz,wght].ttf"
       as="font"
-      type="font/woff2"
+      type="font/ttf"
       crossorigin
     />
     <style>
       @font-face {
         font-family: "Google Sans";
         font-style: normal;
-        src: url("fonts/GoogleSans[GRAD,opsz,wght]") format("ttf");
```

Comment:
The correct format is apparently "truetype", or it just figures it out if I leave it off.